### PR TITLE
dpkg: fix pkg-config file installed to wrong place

### PR DESCRIPTION
**Pull request by ashb**
libdpkg includes a pkg-config file -- it works if I manually specified
PKG_CONFIG_PATH=$(brew --prefix dpkg)/libexec/lib/pkgconfig so this
fixes it to install the .pc file into the right default search path.

### CODE REVIEW

File: Formula/dpkg.rb [hunk 1]
```diff
@@ -40,6 +40,7 @@ def install
 
     bin.install Dir["#{libexec}/bin/*"]
     man.install Dir["#{libexec}/share/man/*"]
+    (lib/"pkgconfig").install Dir["#{libexec}/lib/pkgconfig/*.pc"]
```
Comment (DomT4) [inline Formula/dpkg.rb]: This should probably be a symlink to avoid potentially breaking current workflows.

Comment (ashb) [inline Formula/dpkg.rb]: Oh yes, good call. Will fix and rebase

Comment (ashb) [inline Formula/dpkg.rb]: @DomT4 added as d234b2d

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/3d0c554723a69f0278ad43d3b2d8f4df14f85f51. Thank you for your contribution to Homebrew @ashb; we appreciate it! :smiley_cat:

**Pull request closed without merging**